pages/common/2to3.md

```diff
@@ -1,6 +1,8 @@
 # 2to3
 
 > Automated Python 2 to 3 code conversion.
+> This Module has been depreciated and has been removed since **`Python 3.10`**.
```

Comment:
```suggestion
> This Module has been depreciated and has been removed since `Python 3.10`.
```
What these * are for?
If you tried to use it like in markdown, we do not support this notation. See style guide.